frontend/src/entities/enemies/Guy.js

```diff
@@ -1,8 +1,8 @@
 import Enemy from "./Enemy";
 
 export default class Guy extends Enemy {
-    constructor (x,y,z) {
-        super(x, y, z, 0.2);
+    constructor (x, y, z, scale) {
```

Comment:
Why do you want to pass `scale` into `Guy`?

Reply:
To preload the Guy model, I render one at the beginning of the game below the map at a super small scale. There's probably a better way to preload models, so I'm open to changes.